src/actions/storage-actions.js

```diff
@@ -152,3 +152,18 @@ export const setStorageScenarioAction = (scenario) => {
         type: "SET_STORAGE_SCENARIO",
     };
 };
+
+export const setLuksEncryptionAction = (isEncrypted) => ({
```

Comment:
I think three actions for this is overkill. (too much boilerplate code)
I would suggest to have one action setLuksEncryptionDataAction({ isEncrypted, passphrase, confirmPassphrase }) and make it possible to profide only part of the keys.
If a key is not passed the reducer should not overwrite the existing value. 

What do you think?

Reply:
It would probably be too messy. I have modified the code to only one function.